packages/core/src/forms/control-inline/control-inline.element.ts

```diff
@@ -67,8 +76,29 @@ export class CdsInternalControlInline extends CdsControl {
     `;
   }
 
-  connectedCallback() {
-    super.connectedCallback();
-    this.setAttribute('cds-control-inline', '');
+  firstUpdated(props: Map<string, any>) {
+    super.firstUpdated(props);
+    this.inputControl.addEventListener('change', () => (this.checked = this.inputControl.checked));
+    this.observers.push(
```

Comment:
Do we want to watch for `disabled` as well?

Reply:
Yeah good catch, interestingly I realized clr-angular misses this use case as well it looks like.

Reply:
Fixed